### DISCUSSION TITLE
"The results could not be loaded" error and dynamic fields not loading

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#329] [Q&A]
Hi Franky,

I'm not sure why "The results could not be loaded" error keeps happening. This field is to choose "Country". Even the dynamic fields (on the right side of the screenshot) keep showing a loading icon and they fail to work when I select the number of seats.
![Screenshot 2023-10-23 at 08-04-07 平等心禅修营报名 2023年 11月](https://github.com/liedekef/events-made-easy/assets/101639308/2c24ec80-2242-4d2e-bd5c-4722282a6b31)

The remedies I have tried:
1) Using Varnish Cache on my server, I have already excluded the webpage. There are no other cache plugins on my website.
2) Removed "Block PUT method" in the firewall
![Screenshot 2023-10-23 at 08-12-39 Firewall ‹ Refuge In Dhamma — WordPress](https://github.com/liedekef/events-made-easy/assets/101639308/38a02981-5f47-4bdc-9159-09400ee0b256)

After I purge the cache, everything works but after some time, the errors will come back again.

Any other areas that I missed out to check what is causing these errors?

**Comment by liedekef** [comment]
Either Javascript errors on your page (but unlikely in this case) or a php warning or error in the backend being transferred via the ajax call (most probably). So check your browser network console the moment you change the number of seats and see what answer the webserver returns. And check your php error logfiles.
